netbox_netprod_importer/push.py

```diff
@@ -73,6 +75,7 @@ def __init__(self, netbox_api, hostname, props, *args, overwrite=False,
         self.hostname = hostname
         self.props = props
         self.overwrite = overwrite
+        self.vlan = cachetools.LRUCache(4096)
```

Comment:
You might want to name it differently, to avoid confusion between a possible property and your cache (I sometimes thought during the reviewthat `self.vlan` was the vlan of this device). I would go for something like `self.vlans_cache`.

Reply:
Renamed